assets/airlines/sht.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
This file needs to be reinstated. What’s the reason for deletion? SHT is a commonly used callsign for British Airways domestic flights.

Reply:
Oh. Cancel that commit then.